content/_data/changelogs/weekly.yml

```diff
@@ -6948,6 +6948,97 @@
       authors:
         - jvz
 
+- version: 2.229
+  date: 2020-03-29
+  changes:
+  - type: major bug
+    category: major bug
+    pull: 4602
+    issue: 61688
+    authors:
+    - daniel-beck
+    message: |-
+      Actually load a previously saved global build discarder configuration after restart.
```

Comment:
Would be great to have a better changelog since it goes to LTS backports

Reply:
Attempt at better phrasing is in 2d0e42a0d07e393535a359333296cb20d43651f5 .  Comments, suggestions, and corrections are welcomed.